<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wk</class>
 <widget class="QMainWindow" name="wk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch to Meter Mode</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>861</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Analysis Mode</string>
    </property>
    <widget class="QPushButton" name="meas_setup">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Meas Setup</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trace_setup">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trace Setup</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Trig</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>861</width>
       <height>481</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>L ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>C ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Q ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>D ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>R ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_8"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>X ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_9"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>Z ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_10"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_14">
      <attribute name="title">
       <string>Y ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_11"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_15">
      <attribute name="title">
       <string>Î¸ ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_12"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_16">
      <attribute name="title">
       <string>B ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_13"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_17">
      <attribute name="title">
       <string>G ~ T</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_14"/>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Schoolbook L</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bias</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
